src/transforms/aws_ec2_metadata.rs

```diff
@@ -78,6 +78,11 @@ pub struct Ec2Metadata {
     namespace: Option<String>,
     refresh_interval_secs: Option<u64>,
     fields: Option<Vec<String>>,
+    #[serde(
+        default,
+        skip_serializing_if = "crate::serde::skip_serializing_if_default"
+    )]
+    proxy: ProxyConfig,
```

Comment:
I think we actually need to add this config to the docs for this transform still.